src/components/timecard/select-timecard-period-type/SelectTimecardPeriodType.jsx

```diff
@@ -24,6 +16,17 @@ const SelectTimecardPeriodType = () => {
     shouldFocusError: false,
   });
 
+  const [timePeriods, setTimePeriods] = useState([]);
+  const getResponse = async () => {
+    const response = await getTimePeriodTypes({
+      tenantId: '00000000-0000-0000-0000-000000000000',
+    });
+    setTimePeriods(response.data.items.map((item) => item.name));
+  };
+  useEffect(() => {
+    getResponse();
```

Comment:
Name here is a little vague, perhaps getTimePeriodTypeData?

Reply:
Agreed. Function renamed